source/community.applications/usr/local/emhttp/plugins/community.applications/skins/Narrow/skin.html

```
@@ -117,7 +117,8 @@
 			<div id='sortIconOptions' class='ca_hide'>
 				<button type='button' id="defaultSort" class='sortIcons enabledIcon' data-sortBy='Name' data-sortDir='Up'><?tr("Name Ascending");?></button>
 				<button type='button' class='sortIcons' data-sortBy='Name' data-sortDir='Down'><?tr("Name Descending");?></button>
-				<button type='button' class='sortIcons' data-sortBy='downloads' data-sortDir='Down'><?tr("Downloads");?></button>
+				<button type='button' class='sortIcons' data-sortBy='previousMonthDownloads' data-sortDir='Down'><?tr("Downloads Last Month");?></button>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)